### PR TITLE
Lyft case study

### PR DESCRIPTION
**Pull request by ryan-blunden**
Pages to review:

- [Case study](https://deploy-preview-457--sourcegraph.netlify.com/case-studies/lyft-monolith-to-microservices)
- [Home page carousel](https://deploy-preview-457--sourcegraph.netlify.com/)
- [Blog post](https://deploy-preview-457--sourcegraph.netlify.com/blog/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition)

**NOTE**: Any content changes will require approval from Lyft's comms, legal, and marketing team so while feedback is encouraged, unless you think it should block the posting of the case study, we can address the requested changes in a future PR (typo's are fine).

### CODE REVIEW

File: blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: How Sourcegraph ensured production stability at Lyft during their monolith to microservices decomposition
+author: Ryan Blunden
+publishDate: 2019-12-13T10:00-07:00
+tags: [
+  blog
+]
+slug: production-stability-at-lyft-during-their-monolith-to-microservices-decomposition
+heroImage: https://about.sourcegraph.com/blog/lyft-logo.png
+published: true
+description: "Learn how Sourcegraph code search helped Lyft ensure (largely) issue free production deploys during their monolith to microservices decomposition"
```
Comment (dadlerj) [inline blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md]:
```suggestion
description: "Learn how Sourcegraph code search helped Lyft ensure (largely) issue-free production deploys during their monolith to microservices decomposition"
```

File: blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+<!-- pending approval 
+<p style="text-align: center">
+  <img src="/case-studies/lyft-sourcegraph-case-study-og-embed.jpg" />
+</p> -->
+
+Lyft's mission is to improve people's lives with the world's best transportation. With over 30 million riders in 2018, Lyft's business is growing, as is the number of engineers, repositories, and microservices.
```
Comment (dadlerj) [inline blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md]:
```suggestion
Lyft's mission is to improve people's lives with the world's best transportation. With over 30 million riders in 2018, Lyft's business is growing, as are the numbers of engineers, repositories, and microservices.
```

File: blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Learn how, Sourcegraph's [multi-repository code search](/product/code-search-navigation) helped Lyft ensure (largely) issue free production deploys during their monolith to microservices decomposition.
```
Comment (dadlerj) [inline blogposts/production-stability-at-lyft-during-their-monolith-to-microservices-decomposition.md]:
```suggestion
Learn how Sourcegraph's [multi-repository code search](/product/code-search-navigation) helped Lyft ensure (largely) issue-free production deploys during their monolith to microservices decomposition.
```

File: website/src/pages/case-studies/lyft-monolith-to-microservices.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+const justinPhilips = 'Justin Phillips, Software Engineer, Lyft'
+export default ((props: any) => (
+    <Layout
+        location={props.location}
+        meta={{
+            title:
+                'How Sourcegraph ensured production stability at Lyft during their monolith to microservices decomposition',
+            description:
+                'Learn how Sourcegraph code search helped Lyft ensure (largely) issue free production deploys during their monolith to microservices decomposition',
```
Comment (dadlerj) [inline website/src/pages/case-studies/lyft-monolith-to-microservices.tsx]:
```suggestion
                'Learn how Sourcegraph code search helped Lyft ensure (largely) issue-free production deploys during their monolith to microservices decomposition',
```

File: website/src/pages/case-studies/lyft-monolith-to-microservices.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                    'Sourcegraph code search helped ensure production stability throughout the monolith to microservices decomposition by verifying applications were no longer pointing at the monolith version of a service.',
+                author: justinPhilips,
+                image: '/case-studies/justin-phillips-lyft.jpg',
+            }}
+        >
+            <ContentSection color="white" className="col-md-6">
+                <div className="container">
+                    <p>
+                        Lyft's mission is to improve people's lives with the world's best transportation. With over 30
+                        million riders in 2018, Lyft's business is growing, as is the number of engineers, repositories,
```
Comment (dadlerj) [inline website/src/pages/case-studies/lyft-monolith-to-microservices.tsx]:
```suggestion
                        million riders in 2018, Lyft's business is growing, as are the numbers of engineers, repositories,
```

File: website/src/pages/case-studies/lyft-monolith-to-microservices.tsx (hunk 4 above)
Comment (ryan-blunden) [reply website/src/pages/case-studies/lyft-monolith-to-microservices.tsx]: The word "free" is supporting the word "issue" so I believe the usage of a hyphen here correct.

**Pull request merged**